libs/eo/metering-points/shell/src/lib/eo-metering-points-shell.component.ts

```diff
@@ -24,14 +27,34 @@ import { ChangeDetectionStrategy, Component, NgModule } from '@angular/core';
       :host {
         display: block;
       }
+
+      .grey-color {
+        color: var(--watt-color-neutral-grey-900);
+      }
     `,
   ],
-  template: `<p>You do not have any metering points.</p>`,
+  template: `<ng-container *rxLet="meteringPoints$ as meteringPoints">
+    <p class="grey-color" *ngIf="meteringPoints.length < 1">
```

Comment:
suggestion: Should we make this name more generic, in case one day the color should be blue? :-) And update the class name accordingly.

```suggestion
    <p class="metering-point" *ngIf="meteringPoints.length === 0">
```

Reply:
Funny, i initially had metering-point as a class name, but thought it was a little vague when i just used the class to color the text of the p-container :D

If you like metering-point too, i will make it that name again.